process/sync/metablock.go

```diff
@@ -180,37 +182,32 @@ func (boot *MetaBootstrap) setLastEpochStartRound() {
 func (boot *MetaBootstrap) SyncBlock(ctx context.Context) error {
 	err := boot.syncBlock()
 	if errors.IsGetNodeFromDBError(err) {
-		errSync := boot.syncAccountsDBs()
+		getNodeErr, ok := err.(*errors.GetNodeFromDBErr)
```

Comment:
Misleading names. We have the `if errors.IsGetNodeFromDBError(err) {` instruction and then we can not cast it to a `*errors.GetNodeFromDBErr` ? Let's refactor this.

Reply:
done

Reply:
not done. We have the same function and the same cast. Please refactor.

Reply:
added interface here